.travis.yml

```diff
@@ -10,7 +10,7 @@ services:
 
 matrix:
   include:
-    - env: LANGUAGE=ansible      VERSION=2.5.5-r0 PYTHON_VERSION=2.7.15-r1
+    - env: LANGUAGE=ansible      VERSION=2.5.5 PYTHON_VERSION=2.7.15-r1
```

Comment:
`PYTHON_VERSION` is useless now.

Reply:
Deleted

Reply:
Nope, it's still there in `.travis.yml`, you only removed its use in `travis.py`.